contracts/interfaces/IRibbon.sol

```diff
@@ -10,13 +10,29 @@ interface IStrikeSelection {
     function delta() external view returns (uint256);
 }
 
+interface IPercentStrikeSelection {
+    function getStrikePrice(
+        uint256 expiryTimestamp,
+        bool isPut,
+        uint256 strikeMultiplier
+    ) external view returns (uint256, uint256);
+
+    function delta() external view returns (uint256);
+}
```

Comment:
They should all be united under one interface `IStrikeSelection` rather than being split out. This gives us the ability to be agnostic on what the `getStrikePrice` actually does. We can move the `strikeMultiplier` as a variable on the strike selection contract